src/kibana/components/vislib/lib/data.js

```diff
@@ -73,6 +60,18 @@ define(function (require) {
       }
     }
 
+    Data.prototype._getLabels = function (data) {
+      if (this.type === 'series') {
```

Comment:
every branch of this if resolves to a return, so no need for the `else` statements.`